content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/zebra.rst

```diff
@@ -0,0 +1,235 @@
+=========================
+Zebra label configuration
+=========================
+
+.. |ZPL| replace:: :abbr:`ZPL (Zebra Programming Language)`
+
+In Odoo, labels printed in the Zebra Programming Language (ZPL) file format are designed to fit a
+four-by-six inch label. To resize (or reformat) text to fit a variety of |ZPL| label sizes,
+:ref:`navigate to the ZPL label view <inventory/shipping_receiving/zpl-view>`, and alter the |ZPL|
+code.
+
+.. warning::
+   When customizing code in Odoo, please note that upgrading the database to newer versions may
+   break custom |ZPL| code. **Customers are responsible for maintaining their custom code**.
+
+Refer to the following sections for explanations, and example code, for frequently requested Zebra
+label customizations.
+
+- :ref:`adjust margins <inventory/shipping_receiving/margin>`
+- :ref:`enlarge/minimize barcodes <inventory/shipping_receiving/resize>`
+- :ref:`rotate elements <inventory/shipping_receiving/rotate>`
+
+.. _inventory/shipping_receiving/zpl-view:
+
+Navigate to ZPL label view
+==========================
+
+To begin customizing a Zebra label in Odoo, turn on :ref:`developer mode <developer-mode>`, and on
+the main Odoo dashboard, type `Reports`. From the search results that appear in the resulting pop-up
+window, choose :guilabel:`Settings / Technical / Reporting / Reports` to open the :guilabel:`Reports`
+page.
+
+.. note::
+   To manually navigate to the :guilabel:`Reports` page, go to :menuselection:`Settings app -->
+   Technical --> Reporting: Reports`.
+
+.. image:: zebra/search.png
+   :align: center
+   :alt: Show global search result for "Reports".
+
+On the :guilabel:`Reports` page, in the :guilabel:`Search...` bar, type `ZPL`, and hit "Enter". Upon
+doing so, Odoo presents a list of available Zebra labels in Odoo. Select the desired Zebra label
+from the list to modify it on a separate page.
```

Comment:
Suggestion to use the :kbd: role for pressing keys :) 

Side note, need to look into styling for this!
```suggestion
On the :guilabel:`Reports` page, in the :guilabel:`Search...` bar, type `ZPL`, and hit :kbd:`Enter`.
Upon doing so, Odoo presents a list of available Zebra labels in Odoo. Select the desired Zebra
label from the list to modify it on a separate page.
```